src/main/java/com/acrolinx/client/sdk/utils/BatchCheckIdGenerator.java

```diff
@@ -20,6 +20,7 @@ public static String getId(String integrationShortName)
         String name = "javaSDK";
 
         if (integrationShortName != null && !integrationShortName.isEmpty()) {
+            name = integrationShortName.trim().replace(" ", "-");
```

Comment:
The reason for using `replace` over `replaceAll` is given below:

The underlying implementation of String::replaceAll calls the java.util.regex.Pattern.compile() method each time it is called even if the first argument is not a regular expression. This has a significant performance cost and therefore should be used with care.
When String::replaceAll is used, the first argument should be a real regular expression. If it’s not the case, String::replace does exactly the same thing as String::replaceAll without the performance drawback of the regex.
This rule raises an issue for each String::replaceAll used with a String as first parameter which doesn’t contains special regex character or pattern.